src/Samples/NewWindow.Core/AutoOpen.fs

```diff
@@ -30,7 +30,3 @@ module InOutModule =
     let cata f g = function
       | InOut.In  msg -> msg |> f
       | InOut.Out msg -> msg |> g
```

Comment:
```suggestion
      | InOut.Out msg -> msg |> g

```
Tyson might want a closing newline.

Reply:
It's generally a good idea anyway.

Reply:
Doh!  I thought I had.  Especially important in languages like F# where newlines fill the role `;`s in other languages.

Reply:
This PR still seems to remove the closing newline.